indico/modules/categories/templates/management/_events_list.html

```diff
@@ -0,0 +1,159 @@
+{% from 'pagination.html' import render_pagination %}
+
+{% macro event_row(event) %}
+    <tr class="i-table">
+        <td class="i-table column-icon">
+            <input type="checkbox" name="event_id" value="{{ event.id }}">
+        </td>
+        <td class="i-table column-icon">
+            {% if event.is_self_protected %}
+                <span class="icon-lock"></span>
```

Comment:
`.icon-protection-self` (also, can we rename this to `.icon-protection-protected` or `.icon-protection-self-protected`?! not sure how i missed this name in the PR adding it..)